components/auth/org.wso2.carbon.auth.client.registration/src/main/java/org/wso2/carbon/auth/client/registration/ClientRegistrationDAOException.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.auth.client.registration;
+
+/**
+ *  This is the Exception class for Client Registration DAO related exceptions.
+ */
+public class ClientRegistrationDAOException extends Exception {
```

Comment:
There is a base exception  org.wso2.carbon.auth.core.exception.AuthException that you should inherit all execptions from

Reply:
Fixed with c1cb70daf745fdfb233ec62d8153281286ea98d2